src/main/resources/db/migration/V1.17__Add_Beacon_Note_Table.sql

```diff
@@ -0,0 +1,11 @@
+-- Add beacon_note table to store notes against beacon records
+CREATE TABLE IF NOT EXISTS beacon_note (
```

Comment:
Could this table be named as `note` for consistency with the other comments above